src/lib/ts/conf/tsconfig.ngc.json

```diff
@@ -4,7 +4,8 @@
     "flatModuleId": "AUTOGENERATED",
     "flatModuleOutFile": "AUTOGENERATED",
     "skipTemplateCodegen": true,
-    "strictMetadataEmit": true
+    "strictMetadataEmit": true,
+    "fullTemplateTypeCheck": true
```

Comment:
JFYI, I found some other bugs related to the `strictMetadataEmit` just in case someone opens an issue.

https://github.com/angular/angular/issues/23629
https://github.com/angular/angular/issues/23609